docs/faq/questions/using-cypress-faq.mdx

```diff
@@ -411,7 +411,7 @@ to either:
 - Prevent Cypress from failing the test
 
 This is documented in detail on the
-[Catalog Of Events](/api/events/catalog-of-events) page and the recipe
+[Catalog Of Events](/api/cypress-api/catalog-of-events) page and the recipe
```

Comment:
I think that it makes sense to roll it out all at once with Cy 13, so that we don't have to create multiple versions of the same changes (eg, create an "assertions" section of the new table of contents, then remove it in another PR on another branch). Reorganizing things piecemeal can result in a disjointed user experience; I'd rather have all the changes living in one branch so we can look at the organization holistically.